src/pybind/rados.py

```diff
@@ -1821,16 +1847,204 @@ def lookup_snap(self, snap_name):
 
     def get_last_version(self):
         """
-        Return the version of the last object read or written to.
+            Return the version of the last object read or written to.
```

Comment:
i'd suggest put this change into another commit. but could you enlighten me why you want to indent this doc string? seems we don't indent them in other places.